.bemlintrc.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
https://user-images.githubusercontent.com/38065883/175571995-a049c156-b10e-4fc7-8bbf-da2847117e68.mov

use outline to highlight this form, so your page isn't jumping when user clicks on the button

Reply:
![image](https://user-images.githubusercontent.com/38065883/175572184-f7f24d48-b7f1-466c-a926-43a1f9ea0bd1.png)
Something strange here

Reply:
There are that shouldn't be shown when menu is open

https://user-images.githubusercontent.com/38065883/175572407-b4e32307-8aac-4b76-aa6d-458dd173a6af.mov